coalib/bears/requirements/GoRequirement.py

```diff
@@ -29,3 +30,23 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'go', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> GoRequirement(
+       ...     'github.com/golang/lint/golint', '' , '-u' ).install_command()
+       'go get -u github.com/golang/lint/golint'
+
+       :param return: A string with the installation command.
```

Comment:
eh apparently all the docstrings for this function need to be indented by 1 space, so they are alligned with the ``"""``